src/Javis.jl

```diff
@@ -11,21 +11,45 @@ const LaTeXSVG = Dict{LaTeXString, String}(
 
 include("svg2luxor.jl")
 
-latex(t::LaTeXString) = latex(t, 10, :stroke)
-latex(t::LaTeXString, font_size::Real) = latex(t, font_size, :stroke)
-latex(t::LaTeXString, action::Symbol) = latex(t, 10, action)
+latex(text::LaTeXString) = latex(text, 10, :stroke)
+latex(text::LaTeXString, font_size::Real) = latex(text, font_size, :stroke)
+latex(text::LaTeXString, action::Symbol) = latex(text, 10, action)
 
 """
-    latex(t::LaTeXString, font_size::Real, action::Symbol)
 
-Add the latex string `text` at the current position. 
-The current position is the top left corner of the latex path. This only works if `tex2svg` is installed.
-It can be installed via:
+`latex(text::LaTeXString, font_size::Real, action::Symbol)`
+
+Add the latex string `text` to the top left corner of the LaTeX path. Can be added to `Luxor.jl` graphics such as `Video` or `Drawing`.
+
+**NOTE: This only works if `tex2svg` is installed.**
+**It can be installed using the following command (you may have to prefix this command with `sudo` depending on your installation):**
+
+> `npm install -g mathjax-node-cli`
+
+# Arguments
+- `text::LaTeXString`: a LaTeX string to render.
+- `font_size::Real`: integer font size of LaTeX string. Default `10`.
+- `action::Symbol`: graphics actions defined by `Luxor.jl`. Default `:stroke`. Available actions:
+  - `:stroke` - See `Luxor.strokepath`.
```

Comment:
I would go for: 
- `:stroke` Draws the latex string on the canvas. For more info check `Luxor.strokepath`
- `:path` Creates the path of the latex string but does not render it to the canvas. For more info check [link to the Luxor docs]

Reply:
That's whay I was thinking about doing too. You're too fast of a reviewer! :joy:

Reply:
Haha sorry. I'm trying to give feedback fast such that you don't need to wait.